brands/tourism/hotel.json

```diff
@@ -152,13 +171,13 @@
       "tourism": "hotel"
     }
   },
-  "tourism/hotel|Delta": {
+  "tourism/hotel|Delta Hotels": {
+    "matchNames": ["delta"],
     "tags": {
-      "brand": "Delta",
+      "brand": "Delta Hotels",
       "brand:wikidata": "Q5254663",
       "brand:wikipedia": "en:Delta Hotels",
-      "name": "Delta",
-      "official_name": "Delta Hotels",
+      "name": "Delta Hotels",
```

Comment:
Is each location called “Delta Hotels” or just “Delta Hotel”?

Reply:
I need to look at this further because this is yet another brand 1/2 way through a rebranding

Reply:
Delta Hotels seems to be the dominant branding right now but that could change in 6 months the way the industry is rebranding.